cmd/kube-controller-manager/app/options/options.go

```diff
@@ -202,8 +202,10 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet, allControllers []string, disabled
 	fs.BoolVar(&s.EnableContentionProfiling, "contention-profiling", false, "Enable lock contention profiling, if profiling is enabled")
 	fs.StringVar(&s.ClusterName, "cluster-name", s.ClusterName, "The instance prefix for the cluster")
 	fs.StringVar(&s.ClusterCIDR, "cluster-cidr", s.ClusterCIDR, "CIDR Range for Pods in cluster.")
+	fs.StringSliceVar(&s.ClusterCIDRs, "cluster-cidrs", s.ClusterCIDRs, "list of CIDR Ranges for Pods in cluster.")
```

Comment:
We need to think thru this more.  Today we assume the cluster has a single big CIDR and that per-node CIDRs are carved out from there.  That has proven to be a problem, and we need to enable a cluster to be multiple disjoint CIDRs, and nodes carve from the set of those.  The obvious implementation would be a flag named `cluster-cidrs`.

Would it be cleaner to make this PR add a v6-specific flag?  `--cluster-cidr-ipv6` ?  Or do we need to make this multi-dimensional to indicate which CIDRs are in the same class, and which are not?  E.g. `--cluster-cidrs=v4=10.9.0.0/16,v6=beef::1234::/64` eventually expanding to --cluster-cidrs=v4=10.9.0.0/16,10.99.0.0/16,v6=beef::1234::/64`?